quantization/src/lib.rs

```diff
@@ -15,6 +20,9 @@ pub use encoded_storage::EncodedStorageBuilder;
 pub use encoded_vectors_u8::EncodedQueryU8;
 pub use encoded_vectors_u8::EncodedVectorsU8;
 
+pub use encoded_vectors_pq::EncodedQueryPQ;
+pub use encoded_vectors_pq::EncodedVectorsPQ;
+
 #[derive(Debug)]
 pub struct EncodingError {
     pub description: String,
```

Comment:
Doesn't have to be public because we only use it within this module.

```suggestion
    description: String,
```

Reply:
done